tasks/main.yml

```diff
@@ -27,7 +25,7 @@
       # errors in the exports file to the user.
       # validate: 'exportfs %s'
     notify: export NFS shares
-    when: nfs_server_shares
+    when: (nfs_server_shares | list | length) > 0
```

Comment:
@robled - All I did here was clean up a lint report on this conditional. I'm curious why you've added it at all. If the user passes in an empty list to `nfs_server_shares`, would they not want an empty eports file written out? I'm curious why this line was included to begin with and am curious if it's a good idea to remove it instead of just edit it.

Reply:
It appears that having an empty exports file is OK, based on the fact that it is empty by default at least on some distributions.  Given that, I think this conditional could go away.